requirements-base.txt

```diff
@@ -2,6 +2,7 @@ BeautifulSoup>=3.2.1
 boto3>=1.4.1,<1.4.6
 botocore<1.5.71
 celery>=3.1.8,<3.1.19
+cffi==1.11.5
```

Comment:
Can we make this `>=` for now? In sentry we do not have exact pins yet.

Reply:
cool, updated!